src/components/navigation/left-navigation.vue

```diff
@@ -88,6 +89,9 @@ export default {
             q-tooltip(anchor="center right" self="center left" :content-style="{ 'font-size': '1em' }") {{ $t('navigation.left-navigation.proposals') }}
           q-btn.q-ma-md(:class="{'active': activeTab=== 'members'}" :flat="activeTab !== 'members'" unelevated rounded padding="12px" icon="fas fa-users" size="sm" :color="activeTab === 'members' ? 'primary' : 'disabled'" :to="{ name: 'members' }")
             q-tooltip(anchor="center right" self="center left" :content-style="{ 'font-size': '1em' }") {{ $t('navigation.left-navigation.members') }}
+          q-btn.q-ma-md(:class="{'active': activeTab=== 'election'}" :flat="activeTab !== 'election'" unelevated rounded padding="12px" size="sm" :color="activeTab === 'election' ? 'primary' : 'disabled'" :to="{ name: 'election' }")
+            img.no-active(:class="{ 'active-btn': activeTab=== 'election' }" :src="activeTab=== 'election' ? require('~/assets/icons/voting-icon-white.svg') : require('~/assets/icons/voting-icon.svg')")
```

Comment:
@evgenibir  It would be more readable if you would do the `require` at the top of the file like `const IconVotingWhite = require('~/assets/icons/voting-icon-white.svg')` and then just use IconVotingWhite ( or however you name it )